services/basic_types.proto

```diff
@@ -104,15 +104,6 @@ message AccountID {
          * in that account, without creating anything, and with no creation fee being charged.
          */
         bytes alias = 4;
-
-        /**
-         * The ethereum account 20-byte EVM address to be used initially in place of the public key bytes. This EVM
```

Comment:
This description was taken from the description of the `alias` field above, we should move it back there since we will be expecting EVM address values in the `alias` field.